hashes/src/macros.rs

```diff
@@ -576,6 +576,22 @@ mod test {
         fn all_zeros() -> Self { Self::from_byte_array([0; 32]) }
     }
 
+    #[test]
+    fn macros_work_in_function_scope() {
+        use crate::sha256t;
+
+        sha256t_tag! {
+            #[doc = r" This tests that we can add additional attributes."]
+            pub struct FunctionScopeTag = hash_str("It works");
+        }
+
+        hash_newtype! {
+            /// Some docs.
+            #[doc = r" This tests that we can add additional attributes."]
```

Comment:
This only tests that the attributes are accepted by the macro but not that they are emitted in the macro output.

I suggest you add `#[repr(align(2))]` and then test (can be const, doesn't have to) that the resulting type has alignment 2.

Reply:
Nice idea! Will add.